engine/include/cubos/engine/render/shadows/caster.hpp

```diff
@@ -18,8 +18,9 @@ namespace cubos::engine
     {
         CUBOS_REFLECT;
 
-        float bias = 0.002F;     ///< Shadow bias.
-        float blurRadius = 1.0F; ///< Shadow blur radius.
-        int id = -1;             ///< Caster id for the shadow atlas.
+        float bias = 0.001F;               ///< Shadow bias.
+        float blurRadius = 0.005F;         ///< Shadow blur radius.
```

Comment:
What's reason behind changing this? Does this give better results?

Reply:
I was testing different values of bias to see the results, and I think this bias gives better results. The blurRadius was only for me to see the shadows without much blur to look for shadow acne artifacts.